e2e/test/iothub/service/IoTHubCertificateValidationE2ETest.cs

```diff
@@ -20,7 +22,7 @@ public class IotHubCertificateValidationE2ETest : E2EMsTestBase
         public async Task ServiceClient_QueryDevicesInvalidServiceCertificateHttp_Fails()
         {
             using var sc = new IotHubServiceClient(TestConfiguration.IoTHub.ConnectionStringInvalidServiceCertificate);
-            IotHubCommunicationException exception = await Assert.ThrowsExceptionAsync<IotHubCommunicationException>(
+            var exception = await Assert.ThrowsExceptionAsync<IotHubServiceException>(
```

Comment:
Tests here are skipped currently, so not checking ErrorCode and IotHubStatusCode properties here